.github/workflows/docs.yml

```diff
@@ -0,0 +1,177 @@
+name: Cryptol Docs
+on:
+  push:
+    tags: ["[0-9]+.[0-9]+(.[0-9]+)?"]
+    branches: [master, "release-**"]
+  pull_request:
+  workflow_dispatch:
+
+concurrency:
+  # Only run one of these at a time because they update the global pages.  Don't
+  # cancel existing runs
+  group: "cryptoldoc"
+  cancel-in-progress: false
+
+jobs:
+  # There are two jobs: building docs for this branch, and then
+  # building a full set of docs for and update the public interface.
+  build-branch-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: false
+          fetch-depth: 0
+      - uses: cachix/install-nix-action@v20
+        with:
+          nix_path: nixpkgs=channel:nixos-23.05
+      - name: build PDF docs
+        if: github.event.pull_request.head.repo.fork == false
+        uses: docker://pandoc/latex:2.9.2
+        with:
+          args: >-
+            sh -c
+            "
+            apk add make;
+            tlmgr install subfigure lastpage preprint adjustbox nag collectbox sectsty todonotes palatino mathpazo;
+            cd docs;
+            make
+            "
+      - name: build HTML docs
+        # This builds the docs for the current checked-out branch to verify they
+        # are buildable.
+        shell: bash
+        run: |
+          cd docs/RefMan
+          nix-shell \
+            -p 'python3.withPackages (pp: [pp.sphinx pp.sphinx_rtd_theme])' \
+            --run 'make html'
+  build-pages-docs:
+    runs-on: ubuntu-latest
+    # The public interface should then allow the user to browse the cryptol
+    # documentation at the master branch, but also the documentation associated
+    # with each pull request and each tag (where a tag typically represents a
+    # released version).
+    #
+    # The general process is to checkout a version of the repository at each
+    # pull-request branch and each tag.  Then the html documentation is generated
+    # for each (a more complicate scheme would access artifacts of the doc builds
+    # from those individual builds, but building the docs is relatively fast, so
+    # it's simpler to do it here).  A top-level index file is created that allows
+    # switching between the various versions of the documentation.  Then the
+    # final results are committed to the gh-pages branch so that the
+    # https://cryptol.github.io/ is useable to browse that documentation.
```

Comment:
Perhaps say "the https://cryptol.github.io/ page" or just "https://cryptol.github.io/" rather than "the https://cryptol.github.io/".

Reply:
Fixed.  It was also the wrong URL... :-(